setup_xcode_cpp14.sh

```diff
@@ -3,7 +3,7 @@
 mkdir -p build
 pushd build
 
-conan install .. --build=missing -s build_type=Debug
+conan install .. --build=missing -s build_type=Debug -o testing=True -s compiler.cppstd=14
```

Comment:
If I don't specify the compiler it defaults to `gnu98`. How can we specify the default?